apps/confidential/confidential.go

```diff
@@ -224,6 +227,13 @@ func WithHTTPClient(httpClient ops.HTTPClient) Option {
 	}
 }
 
+// SendX5C specifies if x5c claim(public key of the certificate) should be sent to STS to enable Subject Name Issuer Authentication.
```

Comment:
We adopted a "With" prefix for options.  So this will need to be WithSendX5C().  Though, if we aren't going to do the SNI name, then WithX5C() would be better.